code/modules/materials/definitions/solids/materials_solid_wood.dm

```diff
@@ -43,7 +43,7 @@
 
 /decl/material/solid/organic/wood/generate_recipes(var/reinforce_material)
 	. = ..()
-	if(reinforce_material)	//recipes below don't support composite materials
+	if(holographic || reinforce_material)	//recipes below don't support composite materials
```

Comment:
Maybe slap holographic check in the stack interaction that brings up recepies in first place?

Having to slap it in every single override seems quite prone to people forgetting to do thta and causing issues.